### ISSUE TITLE
Release 0.6.6 changes

### ISSUE THREAD

**Issue by martinholmer** [open-source-economics/Tax-Calculator#841]
I'm having trouble remembering my contributions from one release to another, so I'm opening an issue that I can add to as I work and hopefully others will add to it as their contributions are merged into the master branch.

The last commit included in release 0.6.5 was #820.

**Comment by martinholmer** [comment]
Since release 0.6.5 (#820) I have contributed the following enhancements to Tax-Calculator.
```
Revise code to use smaller puf.csv input file and make changes to create that input file
Remove many debugging variables from functions.py reducing execution time by 42 percent
Add comments to show one way to use Python debugger to trace Tax-Calculator code
Add tests that confirm zeroing-out CALCULATED_VARS at start leaves results unchanged
```

**Comment by MattHJensen** [comment]
> Revise logic used to estimate behavioral responses to policy reforms [#846, #854, #857]

@talumbau, because of this change, we need a new version of TC on TB soon. Would you like to finish work on #844 first? 

@talumbau and @zrisher, do you think any of your work since #820 should be included in the changelog?

**Comment by talumbau** [comment]
@MattHJensen I pinged appropriate parties on #844. Yes, I would like to get it merged before releasing new version. Also, I looked at any of my commits that got merged in after #820. The main thing was just some fixes when running w/o numba. Probably not worth including in release notes.

**Comment by MattHJensen** [comment]
Ok. Thanks. I think #844 is good to go, too. I did leave a few comments on the PR, but I don't think they need to hold it up.

**Comment by MattHJensen** [comment]
Closing and opening an issue for our next release, 0.6.7. Congrats everyone on the new release and thanks @martinholmer for all of the improvements in this one.

**Comment by MattHJensen** [comment]
Here is a link to the release notes. https://github.com/open-source-economics/Tax-Calculator/releases/tag/0.6.6
